build.gradle

```diff
@@ -16,3 +22,7 @@ android {
         consumerProguardFiles 'consumer-proguard.pro'
     }
 }
+
+dependencies {
+    api 'com.usebutton.merchant:button-merchant:1.0.0'
```

Comment:
Using `api` instead of `implementation` allows the consumer of _this_ Kit also have access to the Button Merchant Library. Is that something we want or do we want to completely abstract away the Merchant Library from the consumer?